main.js

```diff
@@ -170,13 +171,25 @@ const cleanStorage= () => {
   localStorage.clear()
   name.textContent = '[Enter Name]'
   focus.textContent = '[Enter Focus]'
-  city.textContent = '[Enter City]'
+  city.textContent = 'Earth'
   curtainBackground.style.filter = 'blur(10px)'
   grayOrColorIcon.src = 'svg/brightness.svg'
   document.body.style.overflow = 'hidden'
   setTimeout(backgroundFilter, 500)
 }
 
+const setGradient= () => {
```

Comment:
Good idea, friend. But I think it's not entirely justified to include the jQuery library for one function only. I would write this function a native JavaScript.
``` javascript
const setGradient = () => {
  greeting.classList.toggle('gradient-font');
  name.classList.toggle('gradient-font');
  gradient.classList.toggle('active');
}
```

Reply:
I am not a pro in js and thanks for giving a good alternative.
I has removed jquery as per your request